### ISSUE TITLE
incorrect return type of GetUint64()

### ISSUE THREAD

**Issue by GoogleCodeExporter** [byzhang/rapidjson#48]
```
Version 0.1 and trunk

return type of GetUint64() is int64_t.

The correct return type is uint64_t.

```

Original issue reported on code.google.com by `milo...@gmail.com` on 15 Nov 2012 at 2:57

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision r86.
```

Original comment by `milo...@gmail.com` on 15 Nov 2012 at 3:01
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Comment 3 by alex.butum, Today (11 minutes ago)
I see two more potential issues:
1) The code for the GetUint64() member function still uses kInt64Flag instead 
of kUint64Flag

2) The code for SetInt() member function on the "else if (IsUint64())" 
statement uses data_.n.i64 instead of data_.n.u64
```

Original comment by `milo...@gmail.com` on 15 Nov 2012 at 7:14
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
The above 2 additional defeats was fixed by revision r89.
```

Original comment by `milo...@gmail.com` on 15 Nov 2012 at 7:18
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision b7d12d8ab509.
```

Original comment by `milo...@gmail.com` on 6 Jun 2014 at 6:04
